docs/ephemeral-testing-w-notifications.md

```diff
@@ -0,0 +1,206 @@
+
+4) Build and push the image:
+
+```shell
+docker/podman build \
+  -t ${QUAY_REPO_NOTIFICATIONS}:${IMAGE_TAG} \
+  --platform linux/amd64 . \
+  -f docker/Dockerfile.notifications-backend.jvm
+
+docker/podman push ${QUAY_REPO_NOTIFICATIONS}:${IMAGE_TAG}
+```
+
+## Deploy to Ephemeral (All-in-One)
+
+Since the Notifications Backend ClowdApp defines both Inventory API and Relations API as dependencies, deploying Notifications to ephemeral alone will handle deploying all three. By leveraging custom images and a local bonfire config, you can deploy your custom images for Inventory and/or Relations all while deploying Notifications to ephemeral
+
+> NOTE: If desired, you can deploy/redeploy Inventory API and/or Relations API separately from Notifications. This is covered fairly well in the [Internal Kessel Docs](https://cuddly-tribble-gq7r66v.pages.github.io/start-here/getting-started/)
+
+### Setup Local Config
+
+By default, bonfire ships with a config file located under $HOME/.config/bonfire/config.yaml. This config file can be used for configuring your custom deployments
+
+> NOTE: If your config is missing, or you want to start clean: `bonfire config write-default`
+
+The below local config will allow you to control all three services as needed. Each service specifies some form of image parameter; this is where you can set your custom image that has your code changes.
+
+> NOTE: If you do not have changes to test for a particular service, you can comment out the parameter section for the service, or set it to its respective default values
+>
+> Notifications Image/Tag Default: quay.io/cloudservices/notifications-backend:latest
+>
+> Relations Image/Tag Default: quay.io/redhat-services-prod/project-kessel-tenant/kessel-relations/relations-api: latest
+>
+> Inventory Image/Tag Default: quay.io/redhat-services-prod/project-kessel-tenant/kessel-inventory/inventory-api:latest
+
+Update your local config with the below settings:
+
+```yaml
+# $HOME/.config/bonfire/config.yaml
+apps:
+- name: notifications
+  components:
+    - name: notifications-backend
+      host: local
+      repo: /path/to/notifications-backend-cloned-repo
+      path: .rhcicd/clowdapp-backend.yaml
+      parameters:
+        IMAGE: quay.io/username/notifications-backend
+        IMAGE_TAG: your-image-tag
+        NOTIFICATIONS_KESSEL_INVENTORY_ENABLED: true
+        NOTIFICATIONS_KESSEL_RELATIONS_ENABLED: true
+- name: kessel
+  components:
+    - name: kessel-inventory
+      host: local
+      repo: /path/to/inventory-api-cloned-repo
+      path: deploy/kessel-inventory-ephem.yaml
+      parameters:
+        INVENTORY_IMAGE: quay.io/username/inventory-api
+        IMAGE_TAG: your-image-tag
+    - name: kessel-relations
+      host: local
+      repo: /path/to/relations-api-cloned-repo
+      path: deploy/kessel-relations.yaml
+      parameters:
+        RELATIONS_IMAGE: quay.io/username/relations-api
```

Comment:
@tonytheleg I think we will want to specify `INVENTORY_IMAGE` and `IMAGE_TAG` here, rather than the relations image, since we will customise inventory but not relations.